samples/powerstore_v250_k8s_124.yaml

```diff
@@ -27,6 +28,12 @@ spec:
       #- name: external-health-monitor
       #  args: ["--monitor-interval=60s"]
 
+      # Uncomment the following to configure how often external-provisioner polls the driver to detect changed capacity
+      # Configure when the storageCapacity is set as "true"
+      # Allowed values: 1m,2m,3m,...,10m,...,60m etc. Default value: 5m
+      #- name: provisioner
+      #  args: ["--capacity-poll-interval=10m"]
```

Comment:
Although this is commented code but just to be consistent with other default values (how we have kept #  args: ["--monitor-interval=60s"]), can we replace 10 with 5 m ?

Reply:
Yes. Thanks for pointing out. Updated the change.